apps/studio/components/interfaces/Integrations/CronJobs/CronJobScheduleSection.tsx

```diff
@@ -1,107 +1,221 @@
-
+        render={() => {
           return (
             <FormItem_Shadcn_ className="flex flex-col gap-1">
               <FormLabel_Shadcn_>Schedule</FormLabel_Shadcn_>
               <FormControl_Shadcn_>
                 <div className="flex flex-col gap-2">
-                  <Select_Shadcn_ onValueChange={onChangeSelectValue} value={presetValue}>
-                    <FormControl_Shadcn_>
-                      <SelectTrigger_Shadcn_>
-                        <SelectValue_Shadcn_ placeholder="Select a method for the HTTP request" />
-                      </SelectTrigger_Shadcn_>
-                    </FormControl_Shadcn_>
-                    <SelectContent_Shadcn_>
-                      {schedules.map((schedule) => {
-                        return (
-                          <SelectItem_Shadcn_ key={schedule.name} value={schedule.expression}>
-                            {schedule.name}
-                          </SelectItem_Shadcn_>
-                        )
-                      })}
-                    </SelectContent_Shadcn_>
-                  </Select_Shadcn_>
-                  <Input_Shadcn_
-                    {...field}
-                    disabled={field.disabled || presetValue !== 'custom'}
-                    size="large"
-                    autoComplete="off"
-                  />
+                  <FormControl_Shadcn_>
+                    <>
+                      <FormLabel_Shadcn_ className="text-foreground-lighter">
+                        {useNaturalLanguage
+                          ? 'Describe your schedule in words'
+                          : 'Enter a cron expression'}
+                      </FormLabel_Shadcn_>
+                      <Input_Shadcn_
+                        value={inputValue}
+                        placeholder={useNaturalLanguage ? 'E.g. every 5 minutes' : '* * * * *'}
+                        onChange={(e) => {
+                          setInputValue(e.target.value)
+                        }}
+                        onKeyDown={(e) => {
+                          if (e.key === 'Enter') {
+                            e.preventDefault()
+                          }
+                        }}
+                      />
+                    </>
+                  </FormControl_Shadcn_>
+
+                  <div className="flex items-center gap-2 mt-2">
+                    <Switch
+                      checked={useNaturalLanguage}
+                      onCheckedChange={() => {
+                        setUseNaturalLanguage(!useNaturalLanguage)
+                        setInputValue('')
+                        setPresetValue('')
+                        setScheduleString('')
+                      }}
+                    />
+                    <p className="text-sm text-foreground-light">Use natural language</p>
+                  </div>
+
+                  <div className="mt-4">
+                    <p className="text-sm text-foreground-light">Use a preset</p>
```

Comment:
probably don't need this label - i think the buttons communicate clear enough that they're presets